packages/react-core/src/demos/examples/Tabs/GrayTabs.tsx

```diff
@@ -0,0 +1,158 @@
+import React from 'react';
+import {
+  PageSection,
+  PageSectionVariants,
+  Tabs,
+  Tab,
+  TabContent,
+  TabContentBody,
+  TabTitleText,
+  Title,
+  Grid,
+  GridItem,
+  Flex,
+  FlexItem,
+  Card,
+  CardBody,
+  CardHeader
+} from '@patternfly/react-core';
+import DashboardWrapper from '../../examples/DashboardWrapper';
+
+export const GrayTabsDemo = () => {
+  const [activeTabKey, setActiveTabKey] = React.useState(0);
+  const [activeNestedTabKey, setActiveNestedTabKey] = React.useState(10);
+
+  // Toggle currently active tab
+  const handleTabClick = (_, tabIndex) => setActiveTabKey(tabIndex);
```

Comment:
We should be explicit about the types of the event and tabIndex params:
`(_: React.MouseEvent, tabIndex: string)`